patches/tools-licenses.py.patch

```diff
@@ -28,3 +28,13 @@ index 6405b31497a425c68b0486a2653071f23e515671..e957c14c924a7a405bf9e20f3c8e2ffc
              # Add all subdirectories that are not marked for skipping.
              for dir in dirs:
                  dirpath = os.path.join(path, dir)
+@@ -663,7 +666,8 @@ def GenerateCredits(
+     for path in third_party_dirs:
+         try:
+             metadata = ParseDir(path, _REPOSITORY_ROOT)
+-        except LicenseError:
```

Comment:
This code is very unlikely to change since https://bugs.chromium.org/p/chromium/issues/detail?id=39240 has been open for 10 years. That said, if it did change due to this bug being fixed, we would then simply be able to drop this patch hunk entirely.